### PR TITLE
WRQ-8428: Sync up ui/screenshot tests and test cases.

### PR DESCRIPTION
**Pull request by dongsuwon**
Enact-DCO-1.0-Signed-off-by: Dongsu Won (dongsu.won@lgepartner.com)

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [x] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Even though test cases were created ui or screenshot tests, some cases were not commented.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
- Sync up ui/screenshot tests and test cases.

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRQ-8428

### CODE REVIEW

File: tests/screenshot/apps/components/Button.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ const ButtonTests = [
 		textSize: 'large',
 		component: <Button>click me</Button>
 	},
+	// [QWTC-2232] Checking of the expanded button will be confirmed in other test cases(QWTC-1832).
```
Comment (juwonjeong) [inline tests/screenshot/apps/components/Button.js:14]: Do you mean the effect of the button when it is clicked? I think it is not related to the QWTC-2232.

File: tests/screenshot/apps/components/SwitchItem.js [hunk 2]
```diff
@@ -40,6 +40,7 @@ const SwitchItemTests = [
 		<SwitchItem>Focused SwitchItem</SwitchItem>,
```
Comment (juwonjeong) [inline tests/screenshot/apps/components/SwitchItem.js:40]: Do you think we need // [QWTC-2130] above line 40 for step 4?

Comment (dongsuwon) [reply tests/screenshot/apps/components/SwitchItem.js:41]: You are correct. I updated comment for QWTC-2130.

File: tests/screenshot/apps/components/Button.js (hunk 1 above)
Comment (dongsuwon) [reply tests/screenshot/apps/components/Button.js:14]: This comment displays information related to duplicate steps.

**Review by juwonjeong — APPROVED** [review]
LGTM

**Review by juwonjeong — APPROVED** [review]
LGTM

**Pull request merged**